burnman/minerals/other.py

```diff
@@ -12,10 +12,34 @@
 
 from .. import mineral_helpers as helpers
 from ..mineral import Mineral
+from ..solidsolution import SolidSolution
+from ..solutionmodel import *
 
 from .SLB_2011 import periclase, wuestite, mg_perovskite, fe_perovskite
 
-
+class make_endmember(SolidSolution):
```

Comment:
This is typical naming for a mineral or solid solution. See mg_fe_perovskite, for example. Do we want to change all the mineral names?!

As for naming to make it sound like a solution, I'm just using a mechanical solution as an easy way to address the problem without duplicating a whole bunch of code. This class is designed to create a fixed composition Material.